recipe/meta.yaml

```diff
@@ -13,13 +13,11 @@ source:
     - windows_cmake_install_bindir.patch       # [win]
 
 build:
-  number: 0
-  # py36 since it uses vc14 like py35
-  skip: true  # [win and py36]
+  number: 1
   features:
     - vc9   # [win and py27]
     - vc10  # [win and py34]
-    - vc14  # [win and py35]
+    - vc14  # [win and (py35 or py36)]
```

Comment:
I have currently installed py36 and local "conda build ." failed for me, because receipt doesn't have support of it. Don't have other reasons to add this, so this might be not important at all.